### PR TITLE
The security component must be installed to use csrf_token()

### LINKED ISSUE #8816: Unknown Twig function csrf_token
See [symfony-docs/security/csrf_in_login_form.rst](https://github.com/symfony/symfony-docs/blob/4.0/security/csrf_in_login_form.rst)

There's no more csrf_token function included in twig since Symfony 4.0.
Or is there a missing **_composer require_** ?

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #8816.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: security/csrf_in_login_form.rst [hunk 1]
```diff
@@ -16,8 +16,13 @@ for CSRF. In this article you'll learn how you can use it in your login form.
 Configuring CSRF Protection
 ---------------------------
 
-First, make sure that the CSRF protection is enabled in the main configuration
-file:
+First, install the Security component in your project:
+
+.. code-block:: terminal
+
+    $ composer require security
```
Comment (xabbuh) [inline security/csrf_in_login_form.rst]: `security-csrf` is enough

Comment (xabbuh) [reply security/csrf_in_login_form.rst]: However, you need the Form component for the Twig function to be available.

Comment (weaverryan) [reply security/csrf_in_login_form.rst]: So, it should be `composer require security-csrf form`, correct?

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Closing in favor of #8873.

**Pull request closed without merging**